egs2/oniku/svs1/local/prep_segments_from_xml.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python3
```

Comment:
Ditto

Reply:
They're the same. Maybe can use soft-link to use without copying.